src/crl/mod.rs

```diff
@@ -495,10 +495,12 @@ mod private {
 mod tests {
     use alloc::vec::Vec;
 
+    use pki_types::CertificateDer;
+
     use super::*;
-    use crate::{
-        subject_name::GeneralName, verify_cert::PathNode, x509::DistributionPointName, Cert, Error,
-    };
+    use crate::end_entity::EndEntityCert;
+    use crate::verify_cert::PartialPath;
+    use crate::{subject_name::GeneralName, x509::DistributionPointName, Cert, Error};
```

Comment:
nit: while you're tidying imports (thanks) I think this should be split up further:

```suggestion
    use crate::subject_name::GeneralName;
    use crate::x509::DistributionPointName;
    use crate::{Cert, Error};
```